docs/tutorials/basic/data.md

```diff
@@ -5,16 +5,16 @@ program. Most training and inference modules in MXNet accepts data iterators,
 which simplifies this procedure, especially when reading large datasets from
 filesystems. Here we discuss the API conventions and several provided iterators.
 
-Data iterators in MXNet is similar to the iterator in Python. In Python, we can
-use the built-in function `iter` with an iterable object (such as list) to
-return an iterator. For example, in `x = iter([1, 2, 3])` we obtain an iterator
-on the list `[1,2,3]`. If we repeatedly call `x.next()` (`__next__()` for Python
-3), then we will get elements from the list one by one, and end with a
-`StopIteration` exception.
+## MXNet Data Iterator  
+Data Iterators in *MXNet* are similar to Python iterator objects. In Python the function `iter` allows to fetch items sequentially by calling  `next()` on iterablePython collection objects such as a Python `list`. `iter` provides a abstract interface for traverising various types of iterable collections without needing to expose the underlying data structure.  
 
-## Introduction
+In MXNet, __Iterators__ return a batch of data `DataBatch` on each call to `next`.
```

Comment:
Capitalized `Iterators` all over the doc doesn't make sense.
They are data iterators or DataIter. iterator is a generic python concept.